server/routes/api/v1Campaigns.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
```

Comment:
I would like to propose a small change to the way these new files are organized, which is to add some more subfolders. I created an example PR here for reference:

https://github.com/ProgramEquity/amplify/pull/700/files

Reply:
I will checkout the example PR with the proposed file organization and include it in the new commit.